en-us/install-instructions.md

```diff
@@ -72,9 +72,18 @@ A simple installer for vanilla launcher [is coming soon](https://github.com/Fabu
 
 ### Modrinth
 
-There is indeed [an official listing of FO on Modrinth.](https://modrinth.com/modpack/fabulously-optimized) However, it is currently experimental and there is no official launcher yet, so no instructions or support are given.
-
-Meanwhile you can `♡ Follow` the project to get notified of updates 🙂
+Note: Modrinth support is very experimental and expect bugs, issues and missing mods, it isn't very well supported for now but it is usable.
+FO is available on Modrinth, [here](https://modrinth.com/modpack/fabulously-optimized)
+Modrinth has installation instructions for ATLauncher, [here](https://docs.modrinth.com/docs/modpacks/playing_modpacks/).
+You might also want farsight, which can be downloaded [here](https://www.curseforge.com/minecraft/mc-mods/farsight) to have further view distance on servers, sadly this cannot be included in the Modrinth pack because the mod is on Curseforge only.
```

Comment:
This should be a part of the actual instructions below

Reply:
eh should be fine now

Reply:
You did not include Farsight instructions in the steps below...

Reply:
yeah but there's a wiki page on adding mods so just mentioning that it's nice to add is fine

Reply:
No, it's not "nice to add". It is a part of FO so it should be a part of these instructions as well, for now. 

That's one reason why Modrinth version is experimental in the first place...